pkg/cmd/cli/cmd/helper.go

```diff
@@ -80,3 +80,22 @@ func selectString(s, spec string) bool {
 	}
 	return match
 }
+
+// parseValue parses the value and returns namespace, resource and the
+// value (resource name) itself. The valid syntax is:
+// oc process mytemplate - implicit namespace (current), implicit resource (template)
+// oc process template/mytemplate - implicit namespace (current), explicit resource
+// oc process ns/template/mytemplate - explicit namespace, explicit resource
+// oc process ns//mytemplate - explicit namespace, implicit resource (template)
+func parseValue(v string) (ns, resource, name string) {
```

Comment:
@liggitt believe me I spend couple minutes about that name :-) and I hated parseNamespaceResourceName ;-) (too long and it does not say from what it parses that...)